lib/src/cli.rs

```diff
@@ -1012,7 +1013,7 @@ impl Opt {
         I::Item: Into<OsString> + Clone,
     {
         let mut args = args.into_iter();
-        let first = if let Some(first) = args.next() {
+        let first = match args.next() { Some(first) => {
```

Comment:
Here and I *think* all the other changes this is `cargo fix` being conservative with the drop ordering change. IMO `if let` is easier to read than `match` and so we should not apply these changes.

Reply:
For reference since I had to go find it - https://doc.rust-lang.org/nightly/edition-guide/rust-2024/temporary-if-let-scope.html